server/src/concepts/offer/model/OfferValidation.js

```diff
@@ -0,0 +1,29 @@
+import Joi from "joi-oid";
+
+const validateCreateOffer = (offer) => {
```

Comment:
I do not see an userID in validation. I am thinking right now because we have two options for that:
send userID within createOffer request (in that case you will have to pass userID in validation)
or
we will have to pass token when creating offer (because only registered users can add offers) so when this request will be sending in useCases we can decode this token and get userId
Even second option is better and more safe i think. (in that case it shouldn't be userID in validation, so for now is good)